tests/conftest.py

```diff
@@ -214,20 +214,41 @@ def pytest_addoption(parser):
                      help="Delay between qutebrowser commands.")
     parser.addoption('--qute-profile-subprocs', action='store_true',
                      default=False, help="Run cProfile for subprocesses.")
-    parser.addoption('--qute-bdd-webengine', action='store_true',
-                     help='Use QtWebEngine for BDD tests')
+    parser.addoption('--qute-bdd-backend', action='store',
+                     choices=['webkit', 'webengine'], help='Set backend for BDD tests')
 
 
 def pytest_configure(config):
-    webengine_arg = config.getoption('--qute-bdd-webengine')
-    webengine_env = os.environ.get('QUTE_BDD_WEBENGINE', 'false')
-    config.webengine = webengine_arg or webengine_env == 'true'
-    # Fail early if QtWebEngine is not available
-    if config.webengine:
+    try:
+        # Try to use QtWebkit as the default backend
+        import PyQt5.QtWebKitWidgets
+        config.backend = 'webkit'
+    except ImportError:
+        # Try to use QtWebEngine as a fallback and fail early
+        # if that's also not available
         import PyQt5.QtWebEngineWidgets
+        config.backend = 'webengine'
+
+    webengine_arg = config.getoption('--qute-bdd-backend')
+    webengine_env = os.environ.get('QUTE_BDD_BACKEND')
```

Comment:
Given that those are not webengine-specific anymore, maybe they should be called

```suggestion
    backend_arg = config.getoption('--qute-bdd-backend')
    backend_env = os.environ.get('QUTE_BDD_BACKEND')
```

instead?